fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1947,7 +1947,7 @@ public FragmentExecParams(PlanFragment fragment) {
                 params.setCoord(coordAddress);
                 params.setBackend_num(backendNum++);
                 params.setQuery_globals(queryGlobals);
-                params.setQuery_options(queryOptions);
+                params.setQuery_options(new TQueryOptions(queryOptions));
```

Comment:
If one query has many fragment, you will new many TQueryOptions  objects. Only need new TQueryOptions when `isEnablePipelineEngine`.